crates/bevy_asset/src/asset_server.rs

```diff
@@ -115,16 +133,19 @@ impl AssetServer {
         loaders.push(Arc::new(loader));
     }
 
+    /// Enables change tracking for assets in the asset server.
```

Comment:
I'm not sure this is correct? IIRC this was for hot-reloading? In either case, we need a bit more documentation here, probably with a link.

Reply:
Yes it is for hot reloading, and that's what I meant by change tracking. Given bevy terminologies I think this line as it is might sound like it's for ECS change tracking (in the asset storages)

Reply:
I think I'm going to grab Rob's definition in the latest PR :eyes: 

```rust
/// Enable watching of the filesystem for changes, if support is available, starting from after
/// the point of calling this function.
```